src/Calendar/__stories__/Calendar.stories.js

```diff
@@ -85,6 +88,11 @@ export const dev = () => (
         locale={text('locale', 'en')} />
 );
 
+export const weekDayStart = () => {
+    const _weekDayStart = number('weekDayStart', 1);
+    return <Calendar weekDayStart={_weekDayStart} />;
```

Comment:
Could this adjustable knob instead be part of the `dev` story above, and have a set value for this story for demo purposes? On the docs site, this just appears as a regular calendar.